File: Nasty-Hacks/Nastyhacks.cpp
#include <iostream>

using namespace std;

int main()
{
   int n;
   cin >> n;

   for (int i = 0; i < n; i++)
   {
       int a, b, c;
       cin >> a >> b >> c;
       
       //int ans = max(a, b-c);
       if (a < b-c)
          cout << "advertise" << endl;
       else if (a == b-c)
          cout << "does not matter" << endl;
       else
          cout << "do not advertise" << endl;
   }

  return 0;
}

File: solutions/nastyhacks.py
n = int(input())

answers = []
for _ in range(0, n):
    r, e, c = [int(x) for x in input().split()]
    if e - c == r:
        answers.append("does not matter")
    elif e - c > r:
        answers.append("advertise")
    else:
        answers.append("do not advertise")

for answer in answers:
    print(answer)